slimmer/slimmer.cpp

```diff
@@ -350,15 +366,15 @@ int parsed_main(int argc, char** argv) {
 
       for (auto& jet : event.chsAK4Jets) {
 
-        if (overlap_em(jet) || jet.pt() < 25.0)
+        if (overlap_em(jet) || jet.pt() < 20.0)
           continue;
 
         // Count jets (including forward)
         auto abseta = std::abs(jet.eta());
         output.set_countjets(hbbfile::countjets::fake, jet, abseta);
 
-        if (not jet.loose)
-          continue;
+        // if (not jet.loose)
```

Comment:
@DylanHsu I think we should really require jets to be loose, because they look super fake otherwise

Reply:
ok, we can probably add the `analysis->hbb` case here
https://github.com/sidnarayanan/PandaAnalysis/blob/master/Flat/src/ModulesJets.cc#L69